docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md

```diff
@@ -1,4 +1,10 @@
-# This document lists known breaking changes in Roslyn after .NET 6 all the way to .NET 7.
+**This document lists known breaking changes in Roslyn after .NET 6 all the way to .NET 7.**
+
+<!--
```

Comment:
I'm not sure why this would be a comment that will not get reflected in the final document.

Reply:
I considered that while I was looking at the documents and realized that it seemed to be a directive to those who were compiling changes rather than to the reader who was reviewing them.